### DISCUSSION TITLE
Date Field year as `yy`

### DISCUSSION THREAD

**Discussion by jycouet** [melt-ui/melt-ui#904] [Q&A]
I'm able to play with the `locale` & `granularity`, but was wondering if it's possible to force year with `2-digit`?
Thx for the info

**Comment by huntabyte** [comment]
It's possible but would require you to implement some heavy customization I believe. 

I imagine the use case is something like a card expiration date or similar?

The only issue from a library perspective is how do we determine what the actual beginning of the date is 🤔 possibly through a prop? I'll think on this one a bit and see how feasible it is to implement such.
